doc/source/whatsnew/v1.4.4.rst

```diff
@@ -16,6 +16,7 @@ Fixed regressions
 ~~~~~~~~~~~~~~~~~
 - Fixed regression in :func:`concat` materializing :class:`Index` during sorting even if :class:`Index` was already sorted (:issue:`47501`)
 - Fixed regression in setting ``None`` or non-string value into a ``string``-dtype Series using a mask (:issue:`47628`)
+- Fixed regression in :meth:`loc.__getitem__` with a list of keys causing an internal inconsistency that could lead to a disconnect between ``frame.at[x, y]`` vs ``frame[y].loc[x]`` (:issue:`22372`)
```

Comment:
I would probably move this to bug fix and put reference to the new issue in the regression section

Reply:
Done

Reply:
I would also add a release note for the #47867 issue in the regression section (in the addition to the bugfix release note)

or are you doing this when you add the test?

Reply:
Wanted to do this with the test, but no strong preference

Reply:
sure. merge when ready.